### DISCUSSION TITLE
`transmission-remote --json --list` response explanation

### DISCUSSION THREAD

**Discussion by amalgame21** [transmission/transmission#5162] [Q&A] [answered]
The `--json` flag Return the RPC response as JSON, but I cannot find the meaning of response like `status` with numbers like `4` or `6`,  and `eta` with `-1` value and something more.

Where can I find a full list to explain the meanings of those key value pairs in the output of `--json` ? thanks.

**Comment by garoto** [comment]
https://github.com/transmission/transmission/blob/main/libtransmission/transmission.h#L1418-L1427

**Reply by xloalx** [reply, marked as answer]
The above lines no longer applicable 

    TR_STATUS_STOPPED = 0, /* Torrent is stopped */
    TR_STATUS_CHECK_WAIT = 1, /* Queued to check files */
    TR_STATUS_CHECK = 2, /* Checking files */
    TR_STATUS_DOWNLOAD_WAIT = 3, /* Queued to download */
    TR_STATUS_DOWNLOAD = 4, /* Downloading */
    TR_STATUS_SEED_WAIT = 5, /* Queued to seed */
    TR_STATUS_SEED = 6 /* Seeding */
